CONTRIBUTING.md

```diff
@@ -42,9 +42,8 @@ Contributing to tests:
 ### Debugging Spec Tests
 
 - To fix errors always focus on passing all minimal tests first without running mainnet tests.
-- Spec tests often compare full expected vs actual states in JSON format. To better understand the diff it's convenient to use mocha's option `--inline-diffs`.
```

Comment:
I couldn't find an equivalent to this option.

Reply:
`--inline-diffs` might require a plugin, see related [discord comment](https://discord.com/channels/593655374469660673/1179716009284870194/1179720028547989604). The error output was sufficient for me so far, not sure if it is actually required, but we should confirm that the error output is sufficient to debug specs tests before removing this from docs.

Reply:
I understand the comment as: there is the possibility to create a plugin to adapt the output to our needs. Does this plugin exist at all? cc @nazarhussain 
Also, at least for `beacon-APIS` specs based tests, our logic truncates JSON outputs programmatically so not sure any automatic diff would produce something meaningful.